src/RestApiContext.php

```diff
@@ -225,10 +225,28 @@ private function sendRequest($method, $url, $body = null)
         }
     }
 
-    private function createRequest($method, $url, $body = null)
+    /**
+     * @param string                $method
+     * @param string                $uri    With or without host
+     * @param string|resource|array $body
+     */
+    private function createRequest($method, $uri, $body = null)
     {
-        $this->request = $this->httpClient->createRequest($method, $url, $this->requestHeaders, $body);
+        if (!$this->hasHost($uri)) {
+            $uri = rtrim($this->httpClient->getBaseUrl(), '/') . '/' . ltrim($uri, '/');
```

Comment:
We could use https://packagist.org/packages/jakeasmith/http_build_url but it would introduce a new dependency for almost nothin so :+1: